062-support-tiered-storage.md

```diff
@@ -0,0 +1,126 @@
+#  Support tiered storage natively in Strimzi
+
+## Current situation
+
+Kafka introduced tiered storage feature since 3.6. The feature is in early stage and it’s nice to support tiered storage configuration natively via Strimzi.
+
+## Motivation
+
+Supporting tiered storage via Strimzi allow Strimzi users to enable tiered storage natively with Strimzi config. By allowing to config custom configuration for `RemoteStorageManager`, users are able to adopt tiered storage feature on Strimzi easily.
```

Comment:
I think we need some link to code or official Kafka documentation where readers can get more info about this `RemoteStorageManager`, even because it's the first time you mention it by assuming people already know about it.

Reply:
I add a reference to KIP405 here, hope it helps.